demos/data_process_hpo/app.py

```diff
@@ -29,18 +29,24 @@ def setup():
     def visualize():
         Visualize.setup()
 
+
 def main():
+
     def hello():
 
-        st.image('imgs/data-juicer.png', output_format='png', use_column_width = True)
+        st.image('imgs/data-juicer.png',
+                 output_format='png',
+                 use_column_width=True)
         demo = 'The demo is coming soon😊'
```

Comment:
The demo for hpo is in progress. Need to modify this demo after it's prepared.

Reply:
Yes, after accepting this PR, I will prepare demos on MS and HF

Reply:
[TODO item](https://github.com/orgs/alibaba/projects/9/views/4?pane=issue&itemId=43973897) is added in the project.